localstack/testing/pytest/fixtures.py

```diff
@@ -1136,6 +1137,36 @@ def _is_not_pending():
 """.strip()
 
 
+@pytest.fixture
+def create_apigateway_function(apigateway_client):
```

Comment:
nit: I'd rename this and give the fixture a shorter / more concise name, like `create_rest_api` or `create_apigw_api`, or similar.

Reply:
I followed the `create_lambda_function` pattern, but I agree with a shorter name, for both :)